Code/GraphMol/ROMol.h

```diff
@@ -191,12 +194,18 @@ class ROMol : public RDProps {
   ROMol(const ROMol &other, bool quickCopy = false, int confId = -1) : RDProps() {
     dp_ringInfo = 0;
     initFromOther(other, quickCopy, confId);
+    numBonds = rdcast<unsigned int>(boost::num_edges(d_graph));
   };
   //! construct a molecule from a pickle string
   ROMol(const std::string &binStr);
 
   virtual ~ROMol() { destroy(); };
 
+  //! \name Molecule Name or Title
```

Comment:
Jokes aside: I think if this is going to be here, then the name should logically be an attribute, not a property (for that we should confirm that it'd doesn't make the molecule too much bigger) and that it should definitely be pickled..